src/index.js

```diff
@@ -185,4 +185,8 @@ export {
   useAbsoluteLayout,
   useFlexLayout,
 } from 'react-table';
+export {
+  Skeleton,
+  SkeletonTheme,
```

Comment:
Since we're exposing these components to consumers, let's also add a component page to the documentation website so its discoverable that this component exists (basic examples of how to use it, link to the official `react-loading-skeleton` documentation, etc.). 

If you think it makes sense to treat that as a separate task, though, let me know!

Reply:
Yeah, let's do it as a separate task since we already have a separate github issue for that 😉 I linked it to this PR previously, removed it now, let's do it right after this one gets merged.